src/Shader.js

```diff
@@ -61,7 +61,7 @@ export var Shader = GObject.registerClass({
   }
 },
 
-                                          class Shader extends Shell.GLSLEffect {
+                                          class Shader extends Shell.GLSLEffect{
```

Comment:
To make clang-format happy.

```suggestion
                                          class Shader extends Shell.GLSLEffect {
```